docs.bruno/kalvium asap/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "kalvium asap",
+  "type": "collection"
```

Comment:
Consider adding a description field to provide more context about the collection. [medium]